Directory.Build.targets

```diff
@@ -175,19 +175,18 @@
 
   <Target Name="GenerateReferenceSource">
     <PropertyGroup>
-      <_GenApiExePath>$(ToolsDir)GenApi.exe</_GenApiExePath>
```

Comment:
@wtgodbe we were still using GenAPI from buildtools here.

Reply:
Good catch!